samza-core/src/main/java/org/apache/samza/runtime/LocalApplicationRunner.java

```diff
@@ -1,4 +1,4 @@
-  /*
+/*
```

Comment:
Also, we want to move the createStreamProcessor helper to a StreamProcessor constructor.